pkgs/applications/version-management/git-lfs/default.nix

```diff
@@ -19,17 +19,18 @@ buildGoPackage rec {
 
   subPackages = [ "." ];
 
-  preBuild = ''
+  # fork/exec /build/go-build2056524003/b001/exe/mangen: exec format error
```

Comment:
This means we need to build the mangen command for the current platform. I am not sure we have support for that in goPackages.

Reply:
git-lfs was fixed in https://github.com/NixOS/nixpkgs/pull/177799